src/CreateMiddleware/CreateMiddleware.php

```diff
@@ -41,10 +41,11 @@ public function process(ServerRequestInterface $request, DelegateInterface $dele
     /**
      * @param string $class
      * @param string|null $projectRoot
+     * @param string|null $classSkeleton
      * @return string
      * @throws CreateMiddlewareException
      */
-    public function process($class, $projectRoot = null)
+    public function process($class, $projectRoot = null, $classSkeleton = null)
```

Comment:
Exactly, but self::CONST is used in this class. It's the reason I had to copy the entire class and add my own code.

Anyway, I think it's easier to let it override the class skeleton by injecting it in the process method from a custom command. If you extend the CreateMiddleware class you need to create a custom command anyway.